Sources/FoundationEssentials/FileManager/FileOperations.swift

```diff
@@ -903,6 +911,43 @@ enum _FileOperations {
     }
     #endif
 
+    #if os(WASI)
+    private static func _linkOrCopyFile(_ srcPtr: UnsafePointer<CChar>, _ dstPtr: UnsafePointer<CChar>, with fileManager: FileManager, delegate: some LinkOrCopyDelegate) throws {
```

Comment:
line length readability nit
```suggestion
    private static func _linkOrCopyFile(
        _ srcPtr: UnsafePointer<CChar>, 
        _ dstPtr: UnsafePointer<CChar>, 
        with fileManager: FileManager, delegate: some LinkOrCopyDelegate
    ) throws {
```